distributed/core.py

```diff
@@ -693,6 +705,17 @@ def _measure_tick(self):
                 tick_duration,
             )
         self.digest_metric("tick-duration", tick_duration)
+        self._observed_tick_durations.append((now, tick_duration))
+        # TODO: Do we even care that this is accurate? If the event loop is
+        # blocked for long, we'd store data for a longer period of time. The
+        # stuff below just makes it accurate.
```

Comment:
I think we care about this, otherwise it's hard to define the metric we expose.

Reply:
if we care about semantic accuracy, sure. If we just want an alarm bell, that's not necessarily relevant